scripts/minikube.sh

```diff
@@ -202,6 +211,14 @@ up)
 
     disable_storage_addons
 
+    #  get kubernetes version we are operating on and accordingly enable feature gates
+    KUBE_MAJOR=$(kube_version 1)
+    KUBE_MINOR=$(kube_version 2)
+    if  [ "${KUBE_MAJOR}" -eq 1 ] && [ "${KUBE_MINOR}" -ge 20 ];
```

Comment:
1. not sure you can compare strings with `-eq` or `-ge`
2. no `;` needed when `then` is on the next  line

Reply:
the `-eq or ge` is working as expected, I have removed unwanted `;` in the last refresh.